### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by soullybaby** [sonik-br/GDMENUCardManager#1] [Ideas]
This is a fantastic piece of software, thankyou.

Feature requests:
- It would be great to be able to batch GDI shrink the games that are already on the card, not just the new additions.
- The ability to manage the cover artwork from each game from this interface (to avoid having to extract and rebuild disk images and convert image formats)
- Being able to choose a GDEmu theme from a selection (eg. https://www.reddit.com/r/dreamcast/comments/hhg9e5/my_new_gdmenu_theme_that_im_calling_cleancast/)

**Comment by sonik-br** [comment]
Thanks for using and giving feedback!

For now I don't plan on adding any new feature to it. I have little time to work on it.
Besides there's other tools to do what you asked. ;-)

I use my own tools to apply GDIShrink and to rebuild GDI with updated game artwork (they are basically a front end to other tools).
Can share them as is but I cannot give support for these apps.

About the custom themes you are free to use. The menu files are inside the "tools/menu" folder from my app. You can edit/replace them.
GDMENUCardManager will use what's in the folder when build GDMENU's cdi file.

**Comment by agentadam-code** [comment]
It would be great if a workflow step could be added to see if the game is already on there and to prompt overwrite or skip (along with apply to all). Would save time with bulk drag and drop.
